packages/cli/src/ui/AppContainer.tsx

```diff
@@ -871,6 +872,18 @@ Logging in with Google... Please restart Gemini CLI to continue.
     };
   }, [handleNewMessage]);
 
+  useEffect(() => {
+    if (ctrlCPressCount >= 2) {
```

Comment:
off by 1. this should be > 2, right? It is ok if ctrlC gets pressed twice and we exit after a tiny bit of time. it isn't ok if it gets pressed 3 times without exit.